stdlib/public/Atomics/AtomicMemoryOrderings.swift

```diff
@@ -0,0 +1,287 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
```

Comment:
Remove `2014 - ` from all new files? (SR-7507)

```suggestion
// Copyright (c) 2020 Apple Inc. and the Swift project authors
```